slaves/linux-cross/Dockerfile

```diff
@@ -101,8 +104,18 @@ RUN                                               \
   for f in `ls /x-tools/mipsel-unknown-linux-musl/bin/mipsel-unknown-linux-musl-*`; do     \
     g=`basename $f`;  \
     ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;    \
+  done &&                                        \
+  for f in `ls /x-tools/armv7-unknown-linux-gnueabihf/bin/armv7-unknown-linux-gnueabihf-*`; do     \
+    g=`basename $f`;  \
+    ln -vs $f /usr/bin/`echo $g | sed -e 's/-unknown//'`;    \
   done
 
+# Instruct rustbuild to use the armv7-linux-gnueabihf toolchain instead of the default
+# arm-linux-gnueabihf one
+ENV AR_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-ar \
+  CC_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-gcc \
+  CXX_armv7_unknown_linux_gnueabihf=armv7-linux-gnueabihf-g++
```

Comment:
Yeah the choices of the gcc crate are in theory made to reflect whatever the "standard toolchain" is named for any cross target, but unfortunately most of these targets seem like they don't necessarily have standard toolchains. Right now the "standard" is whatever ubuntu happens to have.

And eh you're right, we'll pick up the others by default anyway. I think the MIPS-MUSL ones won't work by default (but gcc doesn't have a case for that yet), so seems fine to add over time. I'll go ahead and merge!